reference/checkouts.v3.yml

```diff
@@ -400,9 +401,8 @@ paths:
                         name: Chambray Towel
                         url: 'http://store-url.mybigcommerce.com/all/chambray-towel/'
                         quantity: 1
-                        taxable: true
-                        image_url: >-
-                          https://cdn8.bigcommerce.com/s-{store_hash}/products/158/images/382/foglinenbeigestripetowel1b_1024x1024__83011__60806.1534344511.330.500.jpg?c=2
+                        is_taxable: true
+                        image_url: 'https://cdn8.bigcommerce.com/s-{store_hash}/products/158/images/382/foglinenbeigestripetowel1b_1024x1024__83011__60806.1534344511.330.500.jpg?c=2'
                         discounts:
```

Comment:
Please update `discounts` object. `id` should be a string, similar to this:  "id": "3fc2315a-e283-46fb-a4d6-19694ad92d37"